index.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const cors = require("cors");
+
+const menuRouter = require('./routers/menu');
+const orderRouter = require('./routers/order');
+const authRouter = require("./routers/auth");
+
+const app = express();
+const port = process.env.PORT || 4000;
+
+app.use(express.json());
+app.use(cors());
+
+app.use('/menu', menuRouter);
+app.use('/order', orderRouter);
+app.use("/", authRouter);
```

Comment:
nice! idk if you did it on purpose but just in case:
If you're registering a route to the `/` route, its important to put it as the last one. If you had a `/:id` route inside the `authRouter` you are in risk of that route matching for example a GET to `/order` because if would read `order` as the `id`. So to prevent that, do it like this and use it as the last one